src/OV/OpenVarioMenu.cpp

```diff
@@ -414,9 +414,7 @@ int main()
 
   switch (action) {
   case LAUNCH_SHELL:
-    execl("/bin/bash", "bash", "--login", nullptr);
-    execl("/bin/ash", "-ash", nullptr);
-    execl("/bin/ash", "-sh", nullptr);
+    execl("/bin/sh", "sh", "-c", "clear; bash -il", nullptr);
```

Comment:
.... why?? Your commit message does not explain anything.

Reply:
one of the lines is enough to call the shell